### ISSUE TITLE
Feature `table-input` seems broken

### ISSUE THREAD

**Issue by MrChocolatine** [sindresorhus/refined-github#3936]
1. The bug is cause by Refined GitHub

2. URL where the bug appears

Whenever you have a text editor, where you can insert a table like:
- https://github.com/sindresorhus/refined-github/issues/new
- Or simply when replying to this current issue

3. Video

https://user-images.githubusercontent.com/47531779/106594332-dc6bc380-6549-11eb-9b7b-4d3190d95f04.mov

**Comment by fregante** [comment]
It’s not broken, it’s *art*

**Comment by MrChocolatine** [comment]
Dang, if only I knew! Ahah

**Comment by fregante** [comment]
<img width="154" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/106631385-56984a00-6542-11eb-8d22-6c77c00f73de.png">

Works for me on Safari though

**Comment by cheap-glitch** [comment]
I'm not seeing this bug either. Tested the feature in both Firefox and Chromium and it works fine. What browser are you using?

**Comment by MrChocolatine** [comment]
I get it!

After your comments, I decided to try again, slowly but surely.

- I disabled all my extensions, one by one, and tried to insert a table with the button, same result every time. So none of my other extension was the problem.

- I updated my browser to the latest version (I was using the version 86 and the is 88, so not a big gap but still) but same result, so the browser itself was not the problem.

- I created a new blank profile, with just the extension `refined-github` installed and surprise, the button to insert a table was working correctly. The difference here? On my main profile I use a zoom of 90 % on GitHub... If I reset to 100 % the button works as expected... Damn that was tricky.
I don't know how the feature is implemented but is there a way not to depend on users' zoom? I don't even know how it can be related.

---

> I just tested the feature in both Firefox and Chromium and it works fine. What browser are you using?

@cheap-glitch 
I use [Ungoogled-Chromium](https://ungoogled-software.github.io), so basically a Chromium.
